docs/data/charts/bars/BarLabelAbove.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { BarChart } from '@mui/x-charts/BarChart';
+
+export default function BarLabelAbove() {
+  return (
+    <BarChart
+      xAxis={[{ data: ['group A', 'group B', 'group C'] }]}
+      series={[{ data: [4, 3, 5] }, { data: [1, 4.6, 3] }, { data: [2, 5, 6] }]}
```

Comment:
Can we add a negative bar and one bar with the value zero to showcase how the label looks like in those cases?